app/components/market/submit/side-menu.html

```diff
@@ -0,0 +1,45 @@
+<div *ngIf="form && form.value">
+    <div>
+        <h2> Application </h2>
+        <div>
+            <img class="logo" [src]="logo" height="30" width="30" alt="alt" *ngIf="logo">
+            <span> {{ title }} </span>
+        </div>
+    </div>
+    <div>
+        <h2> Action </h2>
+        <div>
+            <span> {{ action }} </span>
+        </div>
+    </div>
+    <div *ngIf="modeState != Modes.None" >
+        <h2> Mode </h2>
+        <div>
+            {{ modeText[modeState] }}
+        </div>
+    </div>
+    <div *ngIf="modeState == Modes.NewPoolAndJob || modeState == Modes.ExistingPoolAndJob">
```

Comment:
make some computed value in the component `showPoolValues` `showJobValues` `showPickedPool`...